rpms/antigen/antigen.spec

```diff
@@ -0,0 +1,29 @@
+Name: antigen
+Version: 2.2.3
+Release: 1%{?dist}
+Summary: A plugin manager for zsh, inspired by oh-my-zsh and vundle.
+
+License: MIT
+URL: https://github.com/zsh-users/antigen
+Source0: https://github.com/zsh-users/antigen/releases/download/v%{version}/v%{version}.tar.gz
+BuildRequires: make
+Requires: zsh
+Requires: git
+
+%description
+Antigen is a small set of functions that help you easily manage your shell (zsh) plugins, called bundles. The concept is pretty much the same as bundles in a typical vim+pathogen setup. Antigen is to zsh, what Vundle is to vim.
+
+%prep
+%setup -q
+
+%install
+make build
```

Comment:
- build in `%build` section.
 - build with `%make_build`
 
But the makefile don't seem to do much work, `make build` could be fine

Reply:
`make build` is fine. but please separate building process into `%build` part
```
%build
make build

%install
make PREFIX=%{buildroot}/%{_prefix} install
```

Reply:
if I have %build, the rpmbuild need debugsourcefiles.list, do I ignore the problem and use --nodebuginfo？

Reply:
`%global debug_package %{nil}`
add to the start of the spec file.

Reply:
done! thx!